cv32e40s/tests/cfg/pmp.yaml

```diff
@@ -0,0 +1,10 @@
+name: pmp
+description: Default pmp configuration for CV32E40S simulations
+compile_flags: >
+  +define+PMP_ENABLE_64
+ovpsim: >
+  --override root/cpu/PMP_registers=64
+  --showoverrides
+cflags: >
+plusargs: >
+cv_sw_march: rv32imc
```

Comment:
I am unsure about this `cv_sw_march` and the `cflags` and `plusargs` above it.
I'll think about it a bit more and discuss with others.
Leaving this comment for now, and will come back to it later.